specs/latest/2.0/index.html

```diff
@@ -697,13 +697,13 @@ <h2 class="no-toc">Table of contents</h2>
   /* Buffer objects */
   // WebGL1:
   void bufferData(GLenum target, GLsizeiptr size, GLenum usage);
-  void bufferData(GLenum target, BufferDataSource? srcData, GLenum usage);
-  void bufferSubData(GLenum target, GLintptr dstByteOffset, BufferDataSource? srcData);
+  void bufferData(GLenum target, ArrayBuffer srcData, GLenum usage);
+  void bufferSubData(GLenum target, GLintptr dstByteOffset, ArrayBuffer srcData);
```

Comment:
Should also be "ArrayBuffer?" to match nullability of WebGL 1.0 -- if that's the direction we're going to go for the preexisting entry points.